conf/application.conf

```diff
@@ -24,6 +24,9 @@ play.filters {
     # The X-Content-Type-Options header. If null, the header is not set.
     contentTypeOptions = "nosniff"
 
+    # The X-Permitted-Cross-Domain-Policies header. If null, the header is not set.
+    permittedCrossDomainPolicies = "master-only"
```

Comment:
The code deployed to staging has the `permittedCrossDomainPolicies` entry removed.  But it is still supplied by the server with the default value.  (Run `curl -i https://webservices-staging.sagebridge.org/` and look at the results.)  I though it would be more explicit if we specify it in `application.conf`.